app/assets/javascripts/application.js

```diff
@@ -5,7 +5,4 @@
 //= require bootstrap-notify
 //= require_tree .
 //= require clipboard
-
-$(document).ready(function() {
-  new Clipboard('#clip');
-});
+//= require clip
```

Comment:
Would name this `clipboard-init` or something like that, to notify that's the `clipboard` and this are connected.